contracts/modules/legacy/LegacyGateway.sol

```diff
@@ -39,13 +39,11 @@ contract LegacyGateway is IGateway, MasterAwareV2 {
   address public _unused_coverMigrator;
 
   IQuotationData public immutable quotationData;
+  INXMToken internal immutable nxmToken;
 
-  constructor(address _quotationData) {
+  constructor(address _quotationData, address _tokenAddress) {
```

Comment:
let's not upgrade these 2 legacy contracts - they will be deprecated soon anyway, not worth it